test-files/ui/features/highlight_features.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Highlight features on click
+
+    Scenario: Start Hootenanny
+        Given I am on Hootenanny
+        And I resize the window
+        And I click Get Started
+
+    Scenario: Add Data
+        And I press "Add Reference Dataset"
+        And I click the "AllDataTypesMerged" Dataset
```

Comment:
I think you need to use the layer "AllDataTypesMergedCucumber" that is created by the conflate_with_stats.feature.  The current dataset is probably local to your dev env.